requirements.txt

```diff
@@ -0,0 +1,10 @@
+BeautifulSoup
+json
```

Comment:
Balíček `json` se instalovat nemusí. Je to jedna ze [standardních knihoven Pythonu](https://docs.python.org/3/library/index.html).